### ISSUE TITLE
Dynamic accent color is forced

### ISSUE THREAD

**Issue by QuantumCoded** [jherkenhoff/qalculate-android#75]
# Expected Behavior
The settings menu should work and have an option to chose dynamic, a color from a pallete, and or a custom RGB color.

Text displayed on light dynamic colors should be inverted to maintain accessibility.

# Current Behavior
There is no setting to change the accent color.

When a light dynamic color is chosen, the text is not inverted to maintain accessibility.

![screenshot](https://github.com/user-attachments/assets/97cf6049-dec5-42c3-9c31-259cb24f756d)

**Comment by jherkenhoff** [comment]
Should be fixed with the new UI overhaul. Do you have the possibility to compile the current main branch and check? Otherwise I will leave this issue open unti the next release in fdroid, so you might confirm that it is fixed.

**Comment by QuantumCoded** [comment]
The contrast issue is indeed fixed in the latest fdroid release! 🎉
